games/Aquaria.yaml

```
@@ -48,7 +48,7 @@
     'false': 1
     'true': 1
 
-  skip_first_vision: false
+  skip_first_vision: true
 
   local_items: ["Energy form"]
```

Since the next Big Async will use Archipelago 0.6.0, it will be necessary to change "Energy form" by "Energy Form" (capital F)
```suggestion
  local_items: ["Energy Form"]
```